packages/shorebird_cli/lib/src/shorebird_cli_command_runner.dart

```diff
@@ -197,6 +199,13 @@ Engine • revision ${shorebirdEnv.shorebirdEngineRevision}''');
     } else {
       try {
         exitCode = await super.runCommand(topLevelResults);
+      } on ProcessExit catch (error, stackTrace) {
+        if (error.exitCode != ExitCode.success.code) {
+          logger
+            ..err('$error')
+            ..detail('$stackTrace');
+        }
```

Comment:
I'm not sure we want this logging here—we previously assumed anything that called `exit` would sufficiently log errors before doing so, and I think that should still hold.